google/cloud/bigtable/data_connection.h

```diff
@@ -60,7 +60,9 @@ struct PrepareQueryParams {
   bigtable::SqlStatement sql_statement;
 };
 
-struct ExecuteQueryParams {};
+struct ExecuteQueryParams {
+  bigtable::BoundQuery const& bound_query;
```

Comment:
Rarely should struct/class members be a reference. Remove the const& for bound_query.